types/splitio.d.ts

```diff
@@ -551,6 +560,9 @@ declare namespace SplitIO {
   interface GaToSplitIntegration {
     type: 'GA_TO_SPLIT',
   }
+  type SPLIT_IMPRESSION = 'IMPRESSION';
+  type SPLIT_EVENT = 'EVENT';
```

Comment:
Do this ones need to be public/exposed? These will change when exposing the constants for clients use anyways, correct?